docs/datadog_dashboard.md

```diff
@@ -0,0 +1,119 @@
+# Datadog Dashboards
+This feature is in beta.
+
+## Overview
+The DatadogDashboard Custom Resource Definition (CRD) allows users to create [dashboards][1] using the Operator and manage them as Kubernetes resources.
+
+## Prerequisites
+
+- Datadog Operator v1.9+
+- **[Helm][2]** for deploying the Datadog Operator
+- **[`kubectl` CLI][3]** for installing a `DatadogDashboard`
+
+Tests were performed on Kubernetes versions >= `v1.30.0`
+
+## Adding a DatadogDashboard
+
+To deploy a `DatadogDashboard` with the Datadog Operator, use the [`datadog-operator` Helm chart][3].
+
+1. Install the [Datadog Operator][4]:
+
+   First, add the Datadog Helm chart with
+
+    ```shell
+    helm repo add datadog https://helm.datadoghq.com
+    ```
+
+    1. Run the following command, substituting your [Datadog API and application keys][5]:
+
+        ```shell
+        helm install my-datadog-operator datadog/datadog-operator --set apiKey=<DATADOG_API_KEY> --set appKey=<DATADOG_APP_KEY> --set datadogDashboard.enabled=true
+        ```
+
+    1. Alternatively, update the [`values.yaml`][6] file of the Datadog Operator Helm chart to include your [Datadog API and application keys][5] and enable `DatadogDashboard`.
+       Then, run
+
+        ```shell
+        helm install my-datadog-operator datadog/datadog-operator -f values.yaml
+        ```
+
```

Comment:
I'm not sure how this renders, but linear steps are clearer. Where possible recommend keeping instructions at the same level (double check my spacing):
````suggestion
## Add a DatadogDashboard

To deploy a `DatadogDashboard` with the Datadog Operator, use the [`datadog-operator` Helm chart][3]. You can install the [Datadog Operator][4], or update an existing Datadog Operator Helm chart.

### Install the Datadog Operator
1. Add the Datadog Helm chart.
     ```shell
     helm repo add datadog https://helm.datadoghq.com
     ```
2. Run the following command, substituting your [Datadog API and application keys][5]:
     ```shell
        helm install my-datadog-operator datadog/datadog-operator --set apiKey=<DATADOG_API_KEY> --set appKey=<DATADOG_APP_KEY> --set datadogDashboard.enabled=true
     ```
### Update the Datadog Operator Helm chart
1. Update the [`values.yaml`][6] file of the Datadog Operator Helm chart to include your [Datadog API and application keys][5].
2. Enable `DatadogDashboard`.
3. Run the following command:
      ```shell
      helm install my-datadog-operator datadog/datadog-operator -f values.yaml
      ```

````

Reply:
thanks @estherk15 ! we're removing this file for now but i will incorporate your suggestions on release in a few weeks.